lib/model/folder_sendrecv.go

```diff
@@ -107,10 +107,10 @@ type sendReceiveFolder struct {
 	pullErrorsMut sync.Mutex
 }
 
-func newSendReceiveFolder(model *model, fset *db.FileSet, ignores *ignore.Matcher, cfg config.FolderConfiguration, ver versioner.Versioner, fs fs.Filesystem) service {
+func newSendReceiveFolder(model *model, fset *db.FileSet, ignores *ignore.Matcher, cfg config.FolderConfiguration, ver versioner.Versioner) service {
 	f := &sendReceiveFolder{
 		folder:        newFolder(model, fset, ignores, cfg),
-		fs:            fs,
```

Comment:
I don't think this change is an improvement. Apparently we don't do that today, but being able to create a folder instance pointing at a filesystem of your choice seems a good idea, while being able to get a filesystem from the database layer is more of an ugly hack due to the need for an mtimefs. This type & constructor doesn't need to know about that hack.

Reply:
Sure, again, please undo the changes and merge if you want this to land in the RC as I won't have time.

Reply:
Yeps!